data_steward/analytics/cdr_ops/combined.py

```diff
@@ -857,3 +858,44 @@ def verify_dataset_labels(dataset):
     result.append(query)
 results = '\nUNION ALL\n'.join(result)
 execute(client, results)
+
+# ## Verify no participant in the pdr_ehr_dup_report list has EHR data.
```

Comment:
@MichaelSchmidt82 One last request, add more to this check description.  When someone reads this, they should know how to identify if the check passes or fails and what to do/check in case of a failure.  This is just updating verbage, but it's important to whoever runs the notebook because they very likely didn't create this check.  You can reference jira issues, lookup tables, etc.  When writing it, think of the information you would want to have if this failed and you didn't write the check.  Add anything useful.

Reply:
@lrwb-aou , this description has been updated.